src/index.coffee

```diff
@@ -40,6 +40,7 @@ tools =
   Polygon: require './tools/Polygon'
   Pan: require './tools/Pan'
   Eyedropper: require './tools/Eyedropper'
+  ShapeSelect: require './tools/ShapeSelect'
```

Comment:
This is pretty nitpicky, but I'd switch the order of this name: `SelectShape`. That way if/when I add `SelectBitmap` they alphabetize together.